src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    partnerName,
+    isMarried,
+    sex,
+  } = person;
+  const partnerStatus = sex === 'f'
+    ? `is my husband`
+    : `is my wife`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">{`${partnerName} ${partnerStatus}`}</p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
```

Comment:
you have the same `<p>` elements here repeating, better move your check inside `<p>` OR would be **much better** if you save needed string to a variable and use it here, try to avoid complex logic in JSX